dvc/tree/s3.py

```diff
@@ -11,12 +11,14 @@
 from dvc.path_info import CloudURLInfo
 from dvc.progress import Tqdm
 from dvc.scheme import Schemes
-from dvc.utils import error_link
+from dvc.utils import conversions, error_link
 
 from .base import BaseTree
 
 logger = logging.getLogger(__name__)
 
+_AWS_CONFIG_PATH = os.path.expanduser("~/.aws/config")
```

Comment:
Does it work on windows? Or this is how awscli does it too?

Reply:
```
and place it in ``~/.aws/credentials`` (or in
``%UserProfile%\.aws/credentials`` on Windows). If you wish to place the
shared credentials file in a different location than the one specified
above, you need to tell aws-cli where to find it. Do this by setting the
appropriate environment variable:
```

Which is what `os.path.expanduser` does on the windows

> On Windows, USERPROFILE will be used if set, otherwise a combination of HOMEPATH and HOMEDRIVE will be used. An initial ~user is handled by stripping the last directory component from the created user path derived above.

Reply:
@isidentical Right, but will the `/` be handled properly? Or do we need to `os.path.join(os.path.expanduser("~"), ".aws", "config")`?

Reply:
Ah completely forgot about that 🤦🏻 fixing it

Reply:
Fixed this, and modified one of the tests to actually use this path.